### PR TITLE
Network work related function update

### PR DESCRIPTION
**Pull request by xutian**
This PR is update for origin PR#1924 created by FengYang.

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1173392 1173393 1172957

**Comment by PyLearner** [comment]
LGTM,
Acked-by: Meng Yang meyang@redhat.com

File: virttest/utils_test/__init__.py [hunk 1]
```diff
@@ -1031,7 +1036,7 @@ def raw_ping(command, timeout, session, output_func):
 def ping(dest=None, count=None, interval=None, interface=None,
          packetsize=None, ttl=None, hint=None, adaptive=False,
          broadcast=False, flood=False, timeout=0,
-         output_func=logging.debug, session=None):
+         output_func=logging.debug, session=None, os_type="linux"):
```
Comment (spcui) [inline virttest/utils_test/__init__.py]: Need to update all test cases and functions calling this func.

**Comment by xutian** [comment]
@spcui 

Updated, Please review. thanks.

**Comment by lmr** [comment]
Hi @xutian. As you might know, we're moving virt-test to avocado-vt, so I please ask you to consider re-implementing this functionality in the later. The new repo is:

https://github.com/avocado-framework/avocado-vt

Please keep in mind the following guidelines when creating the new PR:

1) Most of the time you can pick the patches of your original PR and merge them on a new branch with minimal changes.
3) Don't use autotest APIs, since we're moving away from using them. Please review the code and change things like `utils.system` -> `process.system`, `utils.run` -> `process.run`, so on and so forth.

Thanks, and I'm sorry for any inconveniences we might have caused your team.

**Pull request closed without merging**